wfe/wfe.go

```diff
@@ -327,6 +332,9 @@ func (wfe *WebFrontEndImpl) Directory(ctx context.Context, logEvent *requestEven
 		"new-cert":    newCertPath,
 		"revoke-cert": revokeCertPath,
 	}
+	if features.Enabled(features.AllowKeyRollover) {
+		directoryEndpoints["key-change"] = rolloverPath
```

Comment:
I didn't see an update about Certbot pre-0.6.0 stats - did you ever send it?

However, we may have a better option: How about we check the version string for `Certbot/0.N.`, where N < 6, and don't add the new endpoints if so?